db/seeds/01_users.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+$stdout.puts "1. Creating User..."
```

Comment:
Why not use only `puts` for this and for other places?
```suggestion
puts "1. Creating User..."
```